synfig-studio/src/gui/widgets/widget_waypoint.cpp

```diff
@@ -217,6 +200,7 @@ Widget_Waypoint::Widget_Waypoint(etl::handle<synfig::Canvas> canvas):
 	add(*waypointGrid);
 	add(*interpolationFrame);
 	add(*interpolationGrid);
+	add(*tcbFrame);
 	add(*tcbGrid);
```

Comment:
Why not just place `tcbGrid` inside `tcbFrame`? It's a container too

Reply:
In this [tutorial](https://www.tutorialspoint.com/pygtk/pygtk_frame_class.htm) horizontal box is added to the frame and then the frame is added to the window

Reply:
I’ve split them because I didn’t want to apply that style to the title, but only to the settings under it.

Reply:
@Keyikedalube I've group them together in the last commit.